projects/observability/src/shared/dashboard/data/graphql/topology/topology-data-source.model.ts

```diff
@@ -59,29 +60,15 @@ export class TopologyDataSourceModel extends GraphQlDataSourceModel<TopologyData
 
   @ModelProperty({
     key: 'node-metrics',
-    // tslint:disable-next-line: no-object-literal-type-assertion
-    type: {
-      key: ARRAY_PROPERTY.type,
-      subtype: {
-        key: ModelPropertyType.TYPE,
-        defaultModelClass: EntityMetricAggregationDataSourceModel
-      }
-    } as ArrayPropertyTypeInstance
+    type: PLAIN_OBJECT_PROPERTY.type
   })
-  public nodeMetricSpecifications: MetricAggregationSpecification[] = [];
+  public nodeMetrics!: MetricData;
```

Comment:
Can we give it a better name? We should convert this into a model object too. may be `TopologyMetricsModel` ?

Reply:
We can change the name, 
not sure how much refactoring would be needed to change it to the Model object.

Reply:
Should be minimal.

Reply:
let's rename this and convert it into a model